keychain/derivation.go

```diff
@@ -50,61 +50,76 @@ type KeyFamily uint32
 
 const (
 	// KeyFamilyMultiSig are keys to be used within multi-sig scripts.
-	KeyFamilyMultiSig KeyFamily = 0
+	KeyFamilyMultiSig KeyFamily = iota
```

Comment:
This is very dangerous for values that are stored persistently somewhere. If someone adds a new constant somewhere in the middle (and the reviewers don't notice), all following constants change their value!
That's why we always hard code things that end up in a database (or are communicated over the wire).